docs/how-to/cluster-cfg.rst

```diff
@@ -0,0 +1,60 @@
+Configure cluster_network/ public_network
+=========================================
+
+
+Overview
+--------
+Network configuration is critical for building a high performance Ceph Storage Cluster. Since, Ceph Clients make requests directly to Ceph OSD Daemons. Ceph OSD Daemons perform data replication on behalf of Ceph Clients, which means replication and other factors impose additional loads on Ceph Storage Cluster networks. Refer to `Ceph Network Config <https://docs.ceph.com/en/latest/rados/configuration/network-config-ref/>`_.
+
+Implementation
+--------------
+Microceph cluster config subcommands rely on ‘ceph config’ as the single source of truth for config values and use the same for getting/setting the configs. After updating (setting/resetting) a config value the same request is forwarded to other hosts on the microceph cluster for restarting particular daemons on the host so that updated configs can take effect.
+
+Usage
+-----
+
+The microceph cluster configuration CLI supports, setting, getting, resetting and listing supported config keys (currently 'cluster_network' and 'public_network'). It is designed to be consistent and intuitive.
+
+1. Supported config keys can be configured using the 'set' command:
+
+   .. code-block:: shell
+
+       $ sudo microceph cluster config set cluster_network 10.5.2.165/16
+
+2. Config value for a particular key could be queried using the 'get' command:
+
+   .. code-block:: shell
+
+       $ sudo microceph cluster config get cluster_network
+       +---+-----------------+---------------+
+       | # |       KEY       |     VALUE     |
+       +---+-----------------+---------------+
+       | 0 | cluster_network | 10.5.2.165/16 |
+       +---+-----------------+---------------+
+
+3. A list of all the configured keys can be fetched using the 'list' command:
+
+   .. code-block:: shell
+
+       $ sudo microceph cluster config set public_network 10.5.2.165/16
+       $ sudo microceph cluster config list
+       +---+-----------------+---------------+
+       | # |       KEY       |     VALUE     |
+       +---+-----------------+---------------+
+       | 0 | cluster_network | 10.5.2.165/16 |
+       +---+-----------------+---------------+
+       | 1 | public_network  | 10.5.2.165/16 |
+       +---+-----------------+---------------+
+
+4. Resetting a config key can performed using the 'reset' command:
```

Comment:
Minor suggestion: suggest to add a one-sentence explanation on what resetting means i.e. setting to default values